go.mod

```diff
@@ -15,7 +15,7 @@ require (
 require (
 	github.com/mattermost/logr/v2 v2.0.15
 	github.com/mattermost/mattermost-plugin-api v0.0.27
-	github.com/mattermost/rtcd v0.3.1-0.20220427165117-035048fda047
+	github.com/mattermost/rtcd v0.3.1-0.20220504082845-a64f36fa09dc
```

Comment:
Oh, this will have to be updated once it's merged (you probably know) -- I was stung once by a branch disappearing on me once. :)

Reply:
Yes, I'll probably push a new tag since there are significant changes upstream.